src/inspector/inspector_socket.cpp

```diff
@@ -339,17 +345,24 @@ class WsHandler : public ProtocolHandler {
     }
   }
 
-  void AcceptUpgrade(const std::string& accept_key) override { }
-  void CancelHandshake() override {}
+  void AcceptUpgrade(const std::string& accept_key) override {
+    TRACEV8INSPECTOR_VERBOSE("WsHandler::AcceptUpgrade");
+  }
+  void CancelHandshake() override {
+    TRACEV8INSPECTOR_VERBOSE("WsHandler::CancelHandshake");
+  }
 
   void OnEof() override {
+    TRACEV8INSPECTOR_VERBOSE("WsHandler::OnEof");

```

Comment:
>TRACEV8INSPECTOR_VERBOSE [](start = 4, length = 24)

For my understanding, is there a way we can collect those during Watson crashes?

Reply:
Not sure .. Do we get Office logs with the Watson dump ? Or do we have to go to Cosmos and query for session logs ?

Vlad mentioned long time that there is a process in Windows to enable specific traces in user machines and collect it when needed .. But it was not retrospective collection for sure.